nmclicktests/environment.js

```diff
@@ -25,6 +25,16 @@ Nightmare.action('ug', {
       .wait(1000)
       .then(done.bind(null, null), done);
   },
+  'commitnpush': function(commitMessage, done) {
+    this.wait('.files .file .btn-default')
+      .insert('.staging input.form-control', commitMessage)
+      .wait(100)
+      .click('.commit-grp .dropdown-toggle')
+      .wait(100)
```

Comment:
You can use `ug.click` instead. It already waits.

https://github.com/FredrikNoren/ungit/blob/39a93198f0e53eaf97025ae8d3bb11cf79e161d3/nmclicktests/environment.js#L146-L152